src/MongoDB.Driver.Core/Core/Connections/TcpStreamFactory.cs

```diff
@@ -181,7 +181,7 @@ private async Task ConnectAsync(Socket socket, EndPoint endPoint, CancellationTo
                 try
                 {
                     var dnsEndPoint = endPoint as DnsEndPoint;
-#if NETSTANDARD1_5
+#if NETSTANDARD1_5 || NETSTANDARD2_0
```

Comment:
Just curious why nothing needs to be done in the `Connect` method.

Reply:
Per https://github.com/dotnet/runtime/issues/24917
> This is only happening under specific circumstances:
[1] host is a hostname, such as "localhost", AND [2] KeepAlive is enabled using tcpClient.Client.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.KeepAlive, true)

So we currently bypass this problem by resolving the host for .NET Standard version of our driver and then [calling connect](https://github.com/vincentkam/mongo-csharp-driver/pull/117/files#diff-c823181a4e53905ddcff62eccc5afca7R78) ([async version](https://github.com/vincentkam/mongo-csharp-driver/pull/117/files#diff-c823181a4e53905ddcff62eccc5afca7L92), thereby negating condition [1].